hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/io/FlinkCreateHandle.java

```diff
@@ -42,8 +42,8 @@
  *
  * <p>For the first mini-batch, it initializes and sets up the next file path to write,
  * then closes the file writer. The subsequent mini-batches are appended to a file with new name,
- * the new file would then rename to this file name,
```

Comment:
The subsequent mini-batches are appended to a file => The subsequent mini-batches are written to a file

The docs is actually correct, see `FlinkMergeAndReplaceHandle`.

Reply:
Thansk, Done.